src/LinuxTracing/PerfEventReaders.cpp

```diff
@@ -479,16 +407,21 @@ UprobesWithStackPerfEvent ConsumeUprobeWithStackPerfEvent(PerfEventRingBuffer* r
 
 GenericTracepointPerfEvent ConsumeGenericTracepointPerfEvent(PerfEventRingBuffer* ring_buffer,
                                                              const perf_event_header& header) {
-  perf_event_raw_sample_fixed ring_buffer_record;
-  ring_buffer->ReadRawAtOffset(&ring_buffer_record, 0, sizeof(perf_event_raw_sample_fixed));
+  perf_event_attr flags{};
+
+  flags.sample_type = SAMPLE_TYPE_TID_TIME_STREAMID_CPU;
```

Comment:
Similar to above, Can I ask for a comment here about why SAMPLE_TYPE_TID_TIME_... is the only needed for sample_type?